README.md

```diff
@@ -8,12 +8,10 @@ For examples on how to use these images please take a look at the [documentation
 
 ## 📢 Announcements
 
-We're upgrading to support [dazzle v2](https://github.com/gitpod-io/dazzle) and build with Github Actions.
+We upgraded to support [dazzle v2](https://github.com/gitpod-io/dazzle) and build with GitHub Actions.
 
 Some images may break briefly as part of the upgrade effort. If you have trouble using the `latest` tag, please try `legacy-dazzle-v1`. Think of it as an "escape hatch".
 
-We'll initially release `gitpod/workspace-full`, and then others, updating this page as we go.
```

Comment:
Any reason we remove this line?

Reply:
Forgot to annotate a removal of this line 🙏 

All images were already released, so no information is necessary.